cloudconfig.go

```diff
@@ -0,0 +1,71 @@
+package cloudconfig
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/base64"
+	"html/template"
+	"io/ioutil"
+
+	"github.com/giantswarm/awstpr"
+	tpraws "github.com/giantswarm/awstpr/aws"
+	"github.com/giantswarm/clustertpr/node"
+)
+
+type AWSNode struct {
```

Comment:
The reason why we put AWS specific things here is that we get and decrypt keys from AWS blob store. See 

I can create an interface for operator-specific extensions and make things more abstract.

In case if you folks really really don't want to have operator-specific things in this repo, then we will need to create a separate one and provide some way of providing extensions. I don't have any clear picture of this yes - I will need to think about it today.